_posts/2016-05-16-rust-at-one-year.md

```diff
@@ -0,0 +1,268 @@
+outreach, please let [the community team] know.
+
+[survey]: http://blog.rust-lang.org/2016/05/09/survey.html
+
+[most loved]: https://stackoverflow.com/research/developer-survey-2016
+
+[commits]: https://github.com/rust-lang/rust/commits/master
+[RFCs]: https://github.com/rust-lang/rfcs
+[stability delivered]: http://blog.rust-lang.org/2014/10/30/Stability.html
+[crates]: https://crates.io/
+
+[Bridge]: http://bridgefoundry.org/
+[the community team]: mailto:community-team@rust-lang.org
+
+### Rust in production
+
+This year saw more companies [betting on Rust]. Each one has a story, but two
+particularly resonated.
+
+[betting on Rust]: https://www.rust-lang.org/friends.html
+
+First, there's Dropbox. For the last several years, the company has been
+secretively working on a move
+[away from AWS and onto its own infrastructure][dropbox]. The move, which is now
+complete, included developing custom-build hardware and the software to drive
+it. While much of Dropbox's back-end infrastructure is historically written in
+Go, for some key components the memory footprint and lack of control stood in
+the way of achieving the server utilization they were striving for. They rewrote
+those components in Rust.  In the [words of Jamie Turner][dropbox quote], a lead
+engineer for the project, "the advantages of Rust are many: really powerful
+abstractions, no null, no segfaults, no leaks, yet C-like performance and
+control over memory."
+
+[dropbox]: http://www.wired.com/2016/03/epic-story-dropboxs-exodus-amazon-cloud-empire/
+[dropbox quote]: https://news.ycombinator.com/item?id=11283688
+
+Second, there's Mozilla. They've long been developing [Servo] as a research
+browser engine in Rust, but their first *production* Rust code shipped through a
+different vehicle: *Firefox*. In Firefox 45, without any fanfare, Rust code for
+[mp4 metadata parsing] went out to OSX and 64-bit Linux users; it will hit
+Windows in version 48. The code is currently running in test mode, with its
+results compared against the legacy C++ library: 100% correctness on
+[1 *billion* reported executions][ff]. But this code is just the tip of the iceberg:
+after laying a lot of [groundwork for Rust integration], Firefox is poised to
+bring in significant amounts of new Rust code, including components from
+Servo&mdash;and not just in test mode.
+
+[Servo]: https://github.com/servo/servo/
+[mp4 metadata parsing]: https://github.com/mozilla/mp4parse-rust
+[ff]: https://telemetry.mozilla.org/new-pipeline/dist.html#!cumulative=0&end_date=2016-04-07&keys=__none__!__none__!__none__&max_channel_version=release%252F45&measure=MEDIA_RUST_MP4PARSE_SUCCESS&min_channel_version=null&product=Firefox&sanitize=1&sort_keys=submissions&start_date=2016-03-03&table=0&trim=1&use_submission_date=0
+[groundwork for Rust integration]: http://wiki.mozilla.org/Oxidation
+
+We're hearing [similar stories] from smaller shops that are putting Rust into
```

Comment:
Maybe something like "other shops" or "other companies" to alter the word "smaller"?